doc/examples/effects/lib/main.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
```

Comment:
This is the default boilerplate from `flutter create .`, any reasons to have added this?

Reply:
no, didn't noticed